### ISSUE TITLE
Forge Mod Loader can't find SapManagerPack

### ISSUE THREAD

**Issue by Mrrobin99** [SanAndreasP/ClaySoldiersMod#21]
Hello!

Whenever I try to start this mod with MagicLauncher, the game won't start and I am greeted with a screen that shows this:

Forge Mod Loader has found a problem with your Minecraft installation
The mods and versions listed below could not be found

sapmanpack : minimum version required is 2.4.0

The file ForgeModLoader-client-0.log contains more information

I am using the latest SapManPack (2.4.0) and I have tried disabling all other mods. I'd be grateful if someone would look into this matter.

**Comment by 4everLink** [comment]
I have this same problem right now. this issue was closed without a answer so I'm guessing you figured out the problem and fixed it yourself. mind sharing? because I've tried everything from unchecking all mods but claysoilders and the managerpack in the mods list, to re-arranging their order in the list a number of ways, and making sure I had that "ignore discrepancies" line in the advanced tab. (optifine works just fine right now.)
